.gitignore

```diff
@@ -23,3 +23,4 @@ monicadump.sql
 .phpunit.result.cache
 .composer
 .idea
+.vscode
```

Comment:
```suggestion
```
Please don't add a computer-specific ignore here.
See https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer to know how to add a global ignore file for your needs.